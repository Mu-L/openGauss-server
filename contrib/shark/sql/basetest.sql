create database test_a dbcompatibility 'A';
\c test_a
create extension shark;

\c contrib_regression
create schema basetest;
set search_path = 'basetest';
select N'abc';

create table t1 (a int);
create columnstore index t1_idx1 on t1(a);

CREATE OR REPLACE FUNCTION test RETURNS INT AS
$$
declare
a int;
BEGIN
a := 2;
RETURN a;
END;
$$
LANGUAGE 'pltsql';

create procedure p1
is
begin
null;
end;
/

select l.lanname from pg_language l join pg_proc p on l.oid = p.prolang and p.proname in ('test', 'p1');

drop schema basetest cascade;