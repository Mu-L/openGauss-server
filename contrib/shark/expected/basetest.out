create database test_a dbcompatibility 'A';
\c test_a
create extension shark;
ERROR:  extension "shark" is only supported in D type database
\c contrib_regression
create schema basetest;
set search_path = 'basetest';
select N'abc';
 nvarchar2 
-----------
 abc
(1 row)

create table t1 (a int);
create columnstore index t1_idx1 on t1(a);
NOTICE:  The COLUMNSTORE option is currently ignored
DO $$
BEGIN
create columnstore index on t1 (a);
end $$;
NOTICE:  The COLUMNSTORE option is currently ignored
LINE 3: create columnstore index on t1 (a);
        ^
NOTICE:  The COLUMNSTORE option is currently ignored
CONTEXT:  SQL statement "create columnstore index on t1 (a)"
PL/pgSQL function inline_code_block line 3 at SQL statement
CREATE OR REPLACE FUNCTION test RETURNS INT AS
$$
declare
a int;
BEGIN
a := 2;
RETURN a;
END;
$$
LANGUAGE 'pltsql';
create procedure p1
is
begin
null;
end;
/
select l.lanname from pg_language l join pg_proc p on l.oid = p.prolang and p.proname in ('test', 'p1');
 lanname 
---------
 pltsql
 pltsql
(2 rows)

drop schema basetest cascade;
NOTICE:  drop cascades to 3 other objects
DETAIL:  drop cascades to table t1
drop cascades to function test()
drop cascades to function p1()
